Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -331,17 +331,21 @@ void ABenchmarkGymGameModeBase::TickPlayersConnectedCheck(float DeltaSeconds)
 
 	if (RequiredPlayerCheckTimer.HasTimerGoneOff() && !DeploymentValidTimer.HasTimerGoneOff())
 	{
-		if (SmoothedTotalAuthPlayers < RequiredPlayers)
+		const int32* ActorCount = TotalActorCounts.Find(SimulatedPawnClass);
+		if (ActorCount != nullptr)
```

Comment:
If it doesn't find it, seems like that should be a failure also. Can also do the declaration within the if statement.